packages/app-project/src/screens/ProjectAboutPage/components/AboutDropdownNav/AboutDropdownNav.stories.js

```diff
@@ -31,6 +31,6 @@ export default {
   decorators: [RouterMock]
 }
 
-export const Default = ({ dark }) => <AboutDropdownNav aboutNavLinks={[]} />
```

Comment:
The `aboutNavLinks` array will never be empty because it's populated automatically with `research` and `team` in ProjectAboutPageConnector. Hence using `alwaysNavLinks` here.